app/models/checkout.rb

```diff
@@ -1,7 +1,13 @@
 class Checkout < ApplicationRecord
+  CHECKOUT_PERIOD_IN_DAYS = 14
+
   belongs_to :user
   belongs_to :book
 
   validates :user, presence: true
   validates :book, presence: true, uniqueness: true
+
+  def due_date
+    self.created_at + CHECKOUT_PERIOD_IN_DAYS.day
```

Comment:
Nicely named constant. You can also make this `.days` instead of `.day` which reads a touch more naturally.